resources/language/resource.language.de_de/strings.po

```diff
@@ -326,7 +326,7 @@ msgstr "Beim Weiterspringen pausieren"
 
 msgctxt "#30082"
 msgid "Remember audio / subtitle preferences"
-msgstr ""
+msgstr "Audio- und Untertitel-Einstellungen beibehalten"
```

Comment:
I'm not a German speaker, so please forgive my ignorance, but is "Audio- und" correct or is it a typo?

Reply:
Yes it is correct - it's the short version of "Audioeinstellungen und Untertiteleinstellungen"

Reply:
Well, I learned something! Interesting syntax I couldn't imagine it exists.